### PR TITLE
Issue 40 fix

### PR DESCRIPTION
**Pull request by aditya4196**
Behaviour implemented :-
When in game mode, If screen tab switched or browser minimized, the game will be paused and should enter p to resume the game.

### CODE REVIEW

**Comment by Ramzi-Abidi** [comment]
@aditya4196 Please, try next time to follow branch naming conventions as shown in [contrib-guide](https://github.com/Ramzi-Abidi/Pong/blob/main/CONTRIBUTING.md)

**Comment by aditya4196** [comment]
ohh Sorry, I will follow the process for this Issue if you want.
You can reject the PR and i can create a new feature branch in my repo with my changes and then generate a new PR

File: src/views/MultiplePlayerMode.tsx
Comment (Ramzi-Abidi) [inline src/views/MultiplePlayerMode.tsx:1]: try to add some type safety to the functions

**Review by Ramzi-Abidi — CHANGES_REQUESTED** [review]
@aditya4196 Thanks for the PR but please do the following:

- Try to add some type safety to functions and their return types
- Use proper branch name convention

**Comment by aditya4196** [comment]
@Ramzi-Abidi Sure. I would try to do it by today evening or by tomorrow afternoon.

**Comment by aditya4196** [comment]
@Ramzi-Abidi Just today I was working on this PR, When you say type safety, Are you talking about solution I coded for the Issue 40 or the previous issues solutions. I have tried to define models for every new features I have added for the previous issues.

**Comment by Ramzi-Abidi** [comment]
> @Ramzi-Abidi Just today I was working on this PR, When you say type safety, Are you talking about solution I coded for the Issue 40 or the previous issues solutions. I have tried to define models for every new features I have added for the previous issues.

@aditya4196 Previous is code is fine, just try in this commit to add return types to the function that you wrote (we are using typescript not javascript).

And thanks for your time! 🙂

**Comment by aditya4196** [comment]
Got it! I will get this done tomorrow and submit the PR

**Pull request merged**